artemis-server/src/test/java/org/apache/activemq/artemis/core/persistence/impl/journal/OperationContextUnitTest.java

```diff
@@ -422,4 +422,37 @@ public void onError(final int errorCode, final String errorMessage) {
       Assert.assertEquals(0, operations.get());
    }
 
+   @Test
+   public void testContextWithoutStoreOperationTrackers() {
+      ExecutorService executor = Executors.newSingleThreadExecutor(ActiveMQThreadFactory.defaultThreadFactory(getClass().getName()));
+
+      OperationContextImpl.setMaxStoreOperationTrackers(0);
+      OperationContextImpl context = new OperationContextImpl(executor);
+      Assert.assertNull(context.getStoreOperationTrackers());
+
+      context.storeLineUp();
+      Assert.assertNull(context.getStoreOperationTrackers());
+
+      context.done();
+      Assert.assertNull(context.getStoreOperationTrackers());
+   }
+
+   @Test
+   public void testContextWithStoreOperationTrackers() {
+      ExecutorService executor = Executors.newSingleThreadExecutor(ActiveMQThreadFactory.defaultThreadFactory(getClass().getName()));
+
+      OperationContextImpl.setMaxStoreOperationTrackers(1);
```

Comment:
This sets a static, but it doesnt seem like it is reset later, so does it remain set for all subsequent tests?